source/expressions.tex

```diff
@@ -165,7 +165,7 @@
 \pnum
 \begin{note}
 Historically, lvalues and rvalues were so-called
-because they could appear on the left- and right-hand side of an assignment
+because they can appear on the left- and right-hand side of an assignment
```

Comment:
This is awkward, given the parenthetical. Happy to turn a blind eye, but not for 23.

Reply:
Looks like github lost my comment on this :(
```suggestion
because they appeared on the left- and right-hand side of an assignment
```